src/sentry/cache/redis.py

```diff
@@ -46,17 +53,19 @@ def get(self, key, version=None, raw=False):
 
 
 class RbCache(CommonRedisCache):
-    def __init__(self, **options):
+    def __init__(self, **options: object) -> None:
         cluster, options = get_cluster_from_options("SENTRY_CACHE_OPTIONS", options)
         client = cluster.get_routing_client()
-        CommonRedisCache.__init__(self, client, **options)
+        # XXX: rb does not have a "raw" client -- use the default client
+        super().__init__(client=client, raw_client=client, **options)
 
 
 # Confusing legacy name for RbCache.  We don't actually have a pure redis cache
 RedisCache = RbCache
 
 
 class RedisClusterCache(CommonRedisCache):
-    def __init__(self, cluster_id, **options):
+    def __init__(self, cluster_id: str, **options: object) -> None:
         client = redis_clusters.get(cluster_id)
-        CommonRedisCache.__init__(self, client=client, **options)
+        raw_client = redis_clusters.get(cluster_id, decode_responses=False)
```

Comment:
does this mean that a second connection is opened?

Reply:
that's a good question, I would hope not but I need to dig into the underlying redis library to find out

Reply:
they don't unless they are used -- so in general this won't increase connection count (at least that's how it worked for the other similar change to this!)